src/renderer/wg_engine/tvgWgRenderer.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2023 the ThorVG project. All rights reserved.
+
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include "tvgWgRenderer.h"
+
+// constructor
+WgRenderer::WgRenderer() {
+    initialize();
+}
+
+// destructor
+WgRenderer::~WgRenderer() {
+    release();
+}
+
+// initialize renderer
+void WgRenderer::initialize() {
+    // nothing there
+}
+
+// release renderer
+void WgRenderer::release() {
+    // nothing there
```

Comment:
Just for your information, you can put the LOG instead of comment so that we can figure out any problem later due to the missing implementation. 

for example:
```
TVGERR("WG_ENGINE", "TODO: WgRenderer::release()");
```

It's not mandatory though.

Reply:
got it